mix.exs

```diff
@@ -27,7 +27,8 @@ defmodule ExqScheduler.Mixfile do
   defp deps do
     [
       {:timex, "~> 3.1"},
-      {:redix, ">= 0.0.0"},
+      {:redix, "~> 0.7.0"},
+      {:redix_sentinel, "~> 0.6.0"},
```

Comment:
use `only: :test` for redix_sentinel, it should be the user responsibility to add redix_sentinel to their deps if required.